tests/testcontainers-env/src/main/java/org/alfresco/tas/AlfrescoStackInitializer.java

```diff
@@ -248,6 +276,19 @@ private PostgreSQLContainer createPosgresContainer()
                                              .withStartupTimeout(Duration.ofMinutes(2));
     }
 
+
+    private MySQLContainer createMySqlContainer()
+    {
+        return (MySQLContainer) new MySQLContainer(getImagesConfig().getMySQLImage())
+                                            .withPassword("alfresco")
```

Comment:
might be worth to move to a static variable, seems that it occurs 3 times in this file

Reply:
It already exist for postgre